doc/plots.rst

```diff
@@ -0,0 +1,115 @@
+#####
+Plots
+#####
+
+The **Plots** pane shows the figures and images generated in the Editor, Console or the Variable Explorer during your session, allowing you to interact with them in several ways.
+
+.. image:: images/plots/plots-execution.gif
```

Comment:
I would prefer if this was a bit more representative, showing multiple plots already present and then running your code to generate a new one. This better shows the purpose and basic capabilities of the pane, which is appropriate for a hero GIF.

Reply:
I agree with this suggestion.